src/lib/services/project-service.ts

```diff
@@ -259,16 +260,22 @@ export default class ProjectService {
         return data;
     }
 
-    async updateProject(updatedProject: IProject, user: IUser): Promise<void> {
+    async updateProject(
+        updatedProject: IProjectUpdate,
+        user: IUser,
+    ): Promise<void> {
         const preData = await this.projectStore.get(updatedProject.id);
 
         await this.projectStore.update(updatedProject);
 
+        // updated project contains instructions to update the project but it may not represent a whole project
+        const afterData = await this.projectStore.get(updatedProject.id);
```

Comment:
Without this, our diff is a bit misleading:
![image (24)](https://github.com/Unleash/unleash/assets/455064/9c871654-5287-45bc-98c5-56250605a8d3)